media/css/firefox/accounts-2018.scss

```diff
@@ -0,0 +1,216 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/fonts';
+$image-path: '/media/protocol/img';
+
+@import '../../protocol/css/includes/lib';
+@import '../../protocol/css/components/button';
+@import '../../protocol/css/components/billboard';
+@import '../../protocol/css/components/call-out';
+@import '../../protocol/css/components/feature-card';
+@import '../../protocol/css/components/hero';
+@import '../../protocol/css/templates/card-layout';
+@import '../../protocol/css/components/newsletter-form';
+@import '../../protocol/css/components/modal';
+
+.show {
+    display: block;
+}
+
+#subtitle {
+    @include text-display-md;
+    text-align: center;
+    width: 50%;
+    margin: 0 auto;
+    font-family: $font-open-sans;
+    font-weight: normal;
+}
+
+/* -------------------------------------------------------------------------- */
+// Custom Protocol
+// Hero
+
+.mzp-c-hero.mzp-t-firefox:after {
+    display: none;
+    mask: none;
+}
+
+.mzp-c-call-out {
+    background: transparent;
+}
+
+.mzp-t-firefox {
+    h1, h2, h3, h4 {
+        font-weight: normal;
+    }
+}
+
+// Cards
+
+.mzp-c-card-feature-content {
+    .mzp-c-cta-link {
+        color: #0a84ff;
+        &:hover,
+        &:focus {
+            color: #0060df;
+            border-bottom: 2px solid #0060df;
```

Comment:
$color-link?

Reply:
PS you can look up Protocol colour variables here: https://protocol.mozilla.org/fundamentals/tokens.html